moncli/types.py

```diff
@@ -300,6 +302,72 @@ def _export(self, value):
         return str(value)
 
 
+class PeopleType(MondayType):
+
+    native_type = list
+    native_default = []
+    allow_casts = (int, str, dict)
+    null_value = {}
+    max_allowed = 0
+
+    def __init__(self, id: str = None, title: str = None, max_allowed: int = 0, *args, **kwargs):
+        self.max_allowed = max_allowed
+        if max_allowed > 4 or max_allowed < 0:
+            self.max_allowed = 0
+        if max_allowed == 1:
+            self.native_type = en.cv.PersonOrTeam
+            self.native_default = None
+        super().__init__(id, title, *args, **kwargs)
+
+    def _process(self, value):
+        try:
+            return_list = []
+            if isinstance(value,en.cv.PersonOrTeam):
+                return_list.append(value)
+            elif isinstance(value,list):
+                for data in value:
+                    if isinstance(data, dict):
+                        id = int(data['id'])
+                        kind = enums.PeopleKind[data['kind']]
+                        return_list.append(en.cv.PersonOrTeam(id=id,kind=kind))               
+                    elif isinstance(data, (str,int)):
+                        return_list.append(en.cv.Person(int(data)))
+                    if isinstance(data,en.cv.PersonOrTeam):
+                        return_list.append(data)
+                if self.max_allowed == 1: 
+                    return return_list[0]
+            return return_list
+        except (ValueError, KeyError):
+            raise ConversionError('Cannot convert value "{}" to Person or Team.'.format(data))
+
+
+    def _cast(self, value):
+        try:
+            if isinstance(value, dict):
+                return en.cv.PersonOrTeam(value['id'],enums.PeopleKind[value['kind']])
+            if isinstance(value, (int,str)):
+                return en.cv.Person(int(value))
+        except (ValueError, KeyError):
+            raise ConversionError('Cannot convert value "{}" to Person or Team.'.format(value))
+
+
+    def _process_column_value(self, value: en.cv.ColumnValue):
+        if 'max_people_allowed' in self.metadata:
+            self.max_allowed =  int(self.metadata['max_people_allowed'])
+
+    def _export(self, value):
+        if self.max_allowed == 1:
+            return {'personsAndTeams': [{'id': value[0].id, 'kind': value[0].kind}]}
```

Comment:
This is broken when trying to save.  

The 'kind' needs to be value.kind.name for this to work.

Also, the value in this case will be an individual PersonOrTeam and not a list.  Therefore you do not need the index subscripting.  You should be fine with just value.id.

Please be more thorough with testing.